api/orb/deprecated.go

```diff
@@ -0,0 +1,50 @@
+package orb
+
+import (
+	"github.com/CircleCI-Public/circleci-cli/api"
+	"github.com/CircleCI-Public/circleci-cli/api/graphql"
+	"github.com/pkg/errors"
+)
+
+type deprecatedClient struct {
+	gql *graphql.Client
+}
+
+func (deprecated *deprecatedClient) OrbQuery(configPath string, ownerId string) (*api.ConfigResponse, error) {
```

Comment:
Tiny naming proposal: Could it just be named `client` inside the context of this function?

```suggestion
func (client *deprecatedClient) OrbQuery(configPath string, ownerId string) (*api.ConfigResponse, error) {
```